GPU/Vulkan/PipelineManagerVulkan.cpp

```diff
@@ -164,7 +182,7 @@ static VulkanPipeline *CreateVulkanPipeline(VkDevice device, VkPipelineCache pip
 	rs.lineWidth = 1.0f;
 	rs.rasterizerDiscardEnable = false;
 	rs.polygonMode = VK_POLYGON_MODE_FILL;
-	rs.depthClampEnable = false;
+	rs.depthClampEnable = key.depthClampEnable && gstate_c.Supports(GPU_SUPPORTS_DEPTH_CLAMP);
```

Comment:
I figured it's more efficient to check this on create, rather than on key calculation.  It can mean we have multiple actually identical pipelines on devices which lack support, though.

-[Unknown]